### PR TITLE
[NES-275] Update AggregateToken upgrade script

### PR DESCRIPTION
**Pull request by ungaro**
## What's new in this PR?

Update AggregateToken upgrade script, AggregateToken initial tests.

### CODE REVIEW

File: nest/script/DeployNestContracts.s.sol
Comment (eyqs) [inline nest/script/DeployNestContracts.s.sol:1]: Remove ERC1967Proxy import

File: nest/src/AggregateToken.sol [hunk 1]
```diff
@@ -223,6 +231,41 @@ contract AggregateToken is ComponentToken, IAggregateToken, ERC1155Holder {
...
+        for (uint256 i = 0; i < $.componentTokenList.length; i++) {
+            if ($.componentTokenList[i] == componentToken) {
+                $.componentTokenList[i] = $.componentTokenList[$.componentTokenList.length - 1];
+                $.componentTokenList.pop();
+                break;
+            }
+        }
+
+        emit ComponentTokenUnlisted(componentToken);
+    }
```
Comment (eyqs) [inline nest/src/AggregateToken.sol]: Move this into [N8]

File: nest/script/DeployNestContracts.s.sol
Comment (ungaro) [reply nest/script/DeployNestContracts.s.sol:1]: removed

File: nest/src/AggregateToken.sol (hunk 1 above)
Comment (ungaro) [reply nest/src/AggregateToken.sol]: moved to [N8]

**Review by ungaro — COMMENTED** [review]
resolved

**Comment by ungaro** [comment]
stale branch, do not merge!

**Pull request closed without merging**